blackbox/docs/general/ddl/storage.txt

```diff
@@ -0,0 +1,50 @@
+=========
+ Storage
+=========
+
+How data is stored can be tuned on each column similar to how indexing is defined.
+
+Column Store
+============
+
+Beside of storing the row data as-is (and indexing each value if wanted), each
+value term is stored into a `Column Store`_ by default. The usage of a `Column
+Store`_ is greatly improving global aggregations and groupings and enables
+ordering possiblity as the data for one column is packed at one place.
+
+Turning the `Column Store`_ off will remove the length limitation of 32766 bytes
```

Comment:
I think here is a bit of a leap since the length limitation isn't mentioned before.
Maybe add a short introduction along the lines of "Using the column storage limits the values of string columns to a maximal length of .."